cfme/tests/cli/test_appliance_console_db_restore.py

```diff
@@ -228,10 +237,71 @@ def setup_nfs_samba_backup(appl1):
     nfs_smb.put_file(dump_filename, "{}share.backup".format(loc))
 
 
+def logging_callback(appliance):
+    # TODO (jhenner) Remove me. This is being defined also in some other PR:
+    # Fix minor version update tests. #8521
+    def the_logger(m):
+        logger.info('Appliance %s:\n%s', appliance.hostname, m)
+    return the_logger
+
+
+def restore_db(appl, location=''):
```

Comment:
I think this would be good to define in cfme.utils.appliance.db - there is currently a restore method there using rake.

Reply:
optional/followup ^

Reply:
This will be moved to the `cfme/utils/ssh_expect.py` in the follow-up PRs